lib/configuration/index.js

```diff
@@ -27,6 +27,11 @@ function Config(configuration) {
 
   assign(this, configuration);
 
+  // ability to exclude certain folders
+  if(!Array.isArray(this.excludedPaths)) {
```

Comment:
if we default the value of the input to `false` then `this.excludePaths !== false` should work?